velox/docs/functions/spark/datetime.rst

```diff
@@ -314,6 +314,22 @@ These functions support TIMESTAMP and DATE input types.
 
         SELECT timestamp_millis(1230219000123); -- '2008-12-25 15:30:00.123'
 
+.. spark:function:: timestamp_seconds(x) -> timestamp
+
+    Returns timestamp from the number of seconds (can be fractional) since UTC epoch.
+    Supported types are: TINYINT, SMALLINT, INTEGER, BIGINT, FLOAT, and DOUBLE.
+    For integral types (TINYINT, SMALLINT, INTEGER, BIGINT), the function directly
+    converts the number of seconds to a timestamp. For floating-point types
+    (FLOAT, DOUBLE), the function scales the input to microseconds, truncates
+    towards zero, and saturates the result to the minimum and maximum values of
+    Spark's `timestamp_seconds
+    <https://spark.apache.org/docs/latest/api/sql/index.html#timestamp_seconds>`_.::
```

Comment:
nit: remove the link from function documentation since it's already in the PR description.